tests/p2p/test_p2p.py

```diff
@@ -220,4 +220,4 @@ def test_late_bootstraps(init_session, setup_bootstrap, setup_clients):
                                   findFails=False,
                                   expected=1)
 
-        assert len(hits) == 1, "Could not find new Client bootstrap message"
+        assert len(hits) == 1, "Could not find new Client bootstrap message. client: {0}".format(i[0])
```

Comment:
this is ain't really necessary since we have a message about this client name when query message tries to get it. but it is ok anyway.